libs/candles-chart/src/lib/candles-chart.tsx

```diff
@@ -47,10 +70,48 @@ export const CandlesChartContainer = ({
   const { pubKey } = useVegaWallet();
   const { theme } = useThemeSwitcher();
 
-  const [interval, setInterval] = useState<Interval>(Interval.I15M);
-  const [chartType, setChartType] = useState<ChartType>(ChartType.CANDLE);
-  const [overlays, setOverlays] = useState<Overlay[]>([]);
-  const [studies, setStudies] = useState<Study[]>([Study.VOLUME]);
+  const [storedInterval, setInterval] = useLocalStorageSnapshot(
```

Comment:
suggestion (non blocking): I think its probably worth reaching for  a zustand store with persist middleware for this. It would tidy up the state management, use a single key (which would mean a single read from the store) and allow allow easier default state if nothing is stored